app.js

```diff
@@ -1 +1,165 @@
 'use stict';
+
+// CREATE AN ARRAY OF ALL THE POSSIBLE IMAGES
+var allImages = [];
+// 0
```

Comment:
Instead of numbering each item, consider grouping into blocks of 3-5 objects,  if you delete one object your numbering will be thrown off (will require more time to fix numbering than regrouping).